drizzlepac/haputils/catalog_utils.py

```diff
@@ -2833,7 +2845,7 @@ def _evaluate_segmentation_image(self, segm_img, image_data, big_island_only=Fal
         # The clip eliminates zero-divides when there are no good pixels in the image
         real_pixels = (np.isfinite(image_data) & (image_data != 0)).sum().clip(min=1)
         biggest_source = segm_img.areas.max()/real_pixels
-        log.info("Biggest_source in pixels: %d", segm_img.areas.max())
+        biggest_source_pixels = segm_img.areas.max()
```

Comment:
```suggestion
        biggest_source_pixels = segm_img.areas.max()
        biggest_source = biggest_source_pixels/real_pixels
```
It might be a little clearer to swap these two lines.  Then you can see that the same number is being used in the test.